packages/@aws-cdk/aws-cognito/test/test.user-pool-client.ts

```diff
@@ -0,0 +1,24 @@
+import { expect, haveResourceLike } from '@aws-cdk/assert';
+import cdk = require('@aws-cdk/cdk');
+import { Test } from 'nodeunit';
+import cognito = require('../lib');
+
+export = {
+  'default setup'(test: Test) {
```

Comment:
Would want to test the strategic set-up, too. Like validate that "GenerateSecret" and "EnabledAuthFlow" are passed through correctly.